oidc-rs-spring-boot-web-config/src/main/java/org/entur/auth/spring/config/ConfigAuthManagerResolverAutoConfiguration.java

```diff
@@ -35,39 +36,45 @@
 @EnableConfigurationProperties({EnturAuthProperties.class})
 @RequiredArgsConstructor
 public class ConfigAuthManagerResolverAutoConfiguration {
-    Map<String, AuthenticationManager> authenticationManagers = new HashMap<>();
-    List<JWKSourceWithIssuer> remoteJWKSets = new ArrayList<>();
+    private final Map<String, AuthenticationManager> authenticationManagers = new HashMap<>();
+    private final List<JWKSourceWithIssuer> remoteJWKSets = new ArrayList<>();
 
     private final EnturAuthProperties enturAuthProperties;
     private final AuthProviders authProviders;
-    private final HealthReportListener<
-                    JWKSetSourceWithHealthStatusReporting<SecurityContext>, SecurityContext>
+    private final ObjectProvider<
+                    HealthReportListener<
+                            JWKSetSourceWithHealthStatusReporting<SecurityContext>, SecurityContext>>
             healthReportListener;
 
     @Bean
     public AuthenticationManagerResolver<HttpServletRequest> authenticationManagerResolver() {
         log.debug("Configure AuthenticationManagerResolver");
-        var authoritiesConverter = new TenantJwtGrantedAuthoritiesConverter(authProviders);
+        final var authoritiesConverter = new TenantJwtGrantedAuthoritiesConverter(authProviders);
```

Comment:
Sjekk gjerne om du får erstatte `final var` med `val`
```suggestion
        val authoritiesConverter = new TenantJwtGrantedAuthoritiesConverter(authProviders);
```